ql/src/test/results/clientpositive/druid/druidmini_expressions.q.out

```diff
@@ -1868,9 +1868,9 @@ POSTHOOK: query: SELECT DATE_ADD(cast(`__time` as date), CAST((cdouble / 1000) A
 POSTHOOK: type: QUERY
 POSTHOOK: Input: default@druid_table_alltypesorc
 POSTHOOK: Output: hdfs://### HDFS PATH ###
-1969-02-26	1970-11-04
```

Comment:
those changes seems not correct and might have some issue with timezones

Reply:
checked it out, DATE_ADD(cast(`__time` as date), CAST((cdouble / 1000) changed as the cdouble value changed due to change in rollup. 
Added more columns to this query to make things more clear in this PR